packages/client/hmi-client/src/components/dataset/tera-dataset.vue

```diff
@@ -218,7 +282,7 @@ function highlightSearchTerms(text: string | undefined): string {
 	return text ?? '';
 }
 
-const dataset: Ref<Dataset | null> = ref(null);
+const dataset = ref<Dataset | null>(null);
```

Comment:
Removed type?

Reply:
Collateral damage from something I was testing, but it's the same thing, different syntax